source/release-notes/5.0.txt

```diff
@@ -17,6 +17,26 @@ Release Notes for MongoDB 5.0
 Minor Releases
 --------------
 
+.. _5.0.4-release-notes:
+
+5.0.4 - Upcoming
+~~~~~~~~~~~~~~~~
+
+Issues fixed:
+
+- :issue:`SERVER-60326` Windows Server fails to start when X509 
+  certificate has empty subject name
+- :issue:`SERVER-59876` Large delays in returning from libcrypto.so
+  while establishing egress connections
+- :issue:`SERVER-59456` Start the LDAPReaper threadpool
+- :issue:`SERVER-59226` Deadlock when stepping down with a profile 
+  session marked as uninterruptible
+- :issue:`SERVER-59074` Do not acquire storage tickets just to set/wait
+  on oplog visibility
+
+- `All JIRA issues closed in 5.0.4
+  <https://jira.mongodb.org/issues/?jql=project%20in%20(SERVER%2C%20TOOLS%2C%20WT)%20AND%20resolution%20%3D%20Fixed%20AND%20fixVersion%20%3D%205.0.4>`_
+
```

Comment:
We're missing the bullet for the 5.0.4-changelog.

```
- :ref:`5.0.4-changelog`
```

See the 5.0.3 entry for an example.

Reply:
Ah quite right, sorry I missed that!